docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+    source_name: str
+    target_name: str
+```
+
+| field_name  | data_type | description        | required/optional | example_value   |
+|-------------|-----------|--------------------|-------------------|-----------------|
+| source_name | string    | source column name | required          | "dept_id"       |
+| target_name | string    | target column name | required          | "department_id" |
+
+### transformations
+
+```
+@dataclass
+class Transformation:
+    column_name: str
+    source: str
+    target: str | None = None
+```
+
+| field_name  | data_type | description                                                | required/optional | example_value                    |
+|-------------|-----------|------------------------------------------------------------|-------------------|----------------------------------|
+| column_name | string    | the column name on which the transformation to be applied  | required          | "s_address"                      |
+| source      | string    | the transformation sql expr to be applied on source column | required          | "trim(s_address)" or "s_address" |
+| target      | string    | the transformation sql expr to be applied on source column | required          | "trim(s_address)" or "s_address" |
+
+### thresholds
+
+```
+@dataclass
+class Thresholds:
+    column_name: str
+    lower_bound: str
+    upper_bound: str
+    type: str
+```
+
+| field_name  | data_type | description                                                                                                                                                                  | required/optional                  | example_value      |
+|-------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|--------------------|
+| column_name | string    | the column that should be considered for threshold reconciliation                                                                                                            | required                           | "product_discount" |
+| lower_bound | string    | the lower bound of the difference between the source value and the target value                                                                                              | optional(default=0)                | -5%                | 
+| upper_bound | string    | the upper bound of the difference between the source value and the target value                                                                                              | optional(default=0)                | 5%                 |            
+| type        | string    | user must specify the type of the column. integer is selected for columns of type integer or double, while timestamp is chosen for columns of either date or timestamp types | required("integer" or "timestamp") | "integer"          |
```

Comment:
Should we say that we support all NUMERIC and TEMPORAL types by SQLGLOT ? @sundarshankar89